src/sql/base/browser/ui/table/plugins/autoSizeColumns.plugin.ts

```diff
@@ -20,7 +21,11 @@ export class AutoColumnSize<T> implements Slick.Plugin<T> {
 	private _options: IAutoColumnSizeOptions;
 	private onPostEventHandler = new Slick.EventHandler();
 
-	constructor(options: IAutoColumnSizeOptions = defaultOptions) {
+	constructor(options: IAutoColumnSizeOptions = defaultOptions, configurationService: IConfigurationService) {
+		let configuredMaxWidth = configurationService.getValue<number>('resultsGrid.maxColumnWidth');
```

Comment:
Not sure if we want to bake this into the plugin since this means a result grid config is affecting every table that uses this plugin even outside of the results grid.

Reply:
Yeah I thought about having each user pass it in as the `maxWidth` option but I thought that would make it harder for people using this plugin in the future to get it right in case they forget to pass in the value.

Do you think that's better? If so I'll update the code

Reply:
As it is now, I don't think its correct, perhaps we should have a global `grid` config that would apply to any grid.

Reply:
My worry with that is we don't do auto sizing for most grids so we would either have to or people might be confused about why it's only happening for certain ones.

I've updated the PR to pass in the configured `resultsGrid.maxColumnWidth` as one of the options when creating the auto sizer, so we can choose where to use it. Does that look good?